### PR TITLE
docs - remove statement from INSERT ref page; remove dup para

### PR DESCRIPTION
**Pull request by lisakowen**
received some docs feedback that the INSERT sql ref page includes an incorrect statement related to partitioning.  removed the statement because it is not true for modern syntax.  while reviewing other related ref pages, i found a duplicate paragraph on the UPDATE page.  removed that as well.

### CODE REVIEW

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/markdown/ref_guide/sql_commands/INSERT.html.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ output\_expression
 output\_name
 :   A name to use for a returned column.
 
-To insert data into a partitioned table, you specify the root partitioned table, the table created with the `CREATE TABLE` command. You also can specify a leaf partition in an `INSERT` command. An error is returned if the data is not valid for the specified leaf partition. Specifying a table that is not a leaf partition in the `INSERT` command is not supported. Execution of other DML commands such as `UPDATE` and `DELETE` on any child table of a partitioned table is not supported. These commands must be run on the root partitioned table, the table created with the `CREATE TABLE` command.
+To insert data into a partitioned table, you specify the root partitioned table, the table created with the `CREATE TABLE` command. You also can specify a leaf partition in an `INSERT` command. An error is returned if the data is not valid for the specified leaf partition. Specifying a table that is not a leaf partition in the `INSERT` command is not supported.
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/INSERT.html.md:97]: `Specifying a table that is not a leaf partition in the `INSERT` command is not supported.`
This statement contradicts the fact that someone can INSERT directly on the root table. So we should remove this statement.

File: gpdb-doc/markdown/ref_guide/sql_commands/UPDATE.html.md [hunk 2]
```diff
@@ -94,8 +94,6 @@ For a partitioned table, all of the child tables are locked during the `UPDATE`
 
 In the case of a partitioned table, updating a row might cause it to no longer satisfy the partition constraint of the containing partition. In that case, if there is some other partition in the partition tree for which this row satisfies its partition constraint, then the row is moved to that partition. If there is no such partition, an error will occur. Behind the scenes, the row movement is actually a `DELETE` and `INSERT` operation.
 
-For a partitioned table, all the child tables are locked during the `UPDATE` operation when the Global Deadlock Detector is not enabled \(the default\). Only some of the leaf partitions are locked when the Global Deadlock Detector is enabled. For information about the Global Deadlock Detector, see [Global Deadlock Detector](../../admin_guide/dml.html#topic_gdd).
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/UPDATE.html.md:97]: This is not really a duplicate paragraph. Can you share more context on why this change was requested?

Comment (lisakowen) [reply gpdb-doc/markdown/ref_guide/sql_commands/UPDATE.html.md:97]: this paragraph is a duplicate of the paragraph located two paragraphs previous, i.e. the third paragraph in the Notes section.

**Review by soumyadeep2007 — APPROVED** [review]
